### PR TITLE
[WIP]Label instead of Text

### PR DESCRIPTION
**Pull request by cdikibo**
This is my first pull request for Zulip. This should fix the visibility problem on the night theme on the affected screens. Label replaced Text as a component. Fixes issue #1234

### CODE REVIEW

File: src/start/DevAuthScreen.js [hunk 1]
```diff
@@ -79,11 +79,11 @@ class DevAuthScreen extends PureComponent<Props, State> {
         <View style={styles.container}>
           {progress && <ActivityIndicator />}
           {error && <ErrorMsg error={error} />}
-          <Text style={[styles.field, styles.heading2, inlineStyles.heading]}>Administrators</Text>
+          <Label style={[styles.field, styles.heading2, inlineStyles.heading]}>Administrators</Label>
```
Comment (vishwesh3) [inline src/start/DevAuthScreen.js]: We are using `Label` as `<Label style={styles.text} text="No Internet connection" />`
providing text in `text` props.

Comment (borisyankov) [reply src/start/DevAuthScreen.js]: Also, any string that is added to `Label` is being translated, so don't forget to add these to `messages_en.json`

**Comment by cdikibo** [comment]
So this time, I changed the commit message to Fix:#1234 and I changed how I was using `Label`.  However, I could not find `messages_en.json`  file within the codebase only a reference of `messages.js`. Where is `messages_en.json`  hiding?

**Comment by kunall17** [comment]
@cdikibo Here is your hiding file https://github.com/zulip/zulip-mobile/blob/master/src/i18n/translations/messages_en.json

Press 'T' to redirect to the file finder in GitHub!

**Comment by cdikibo** [comment]
@kunall17 Thank you for showing me where that file was hiding. I was able to add the strings that were being translated to `messages_en.json`.  I believe that I'm finished when it comes to this PR, please let me know if that is not the case @borisyankov & @vishwesh3

**Comment by borisyankov** [comment]
@cdikibo that looks good! Thank you.
Merging it now.

**Pull request merged**